src/list/components/item-fields.js

```diff
@@ -77,13 +77,20 @@ ItemFields.propTypes = {
 export class EditItemFields extends React.Component {
   static get propTypes() {
     return {
+      itemId: PropTypes.string,
       fields: fieldsPropTypes.isRequired,
       onChange: PropTypes.func.isRequired,
     };
   }
 
   componentDidMount() {
-    this._firstField.focus();
+    // Focus the first editable field...
+    if (this.props.fields.itemId) {
+      // For existing items, the first field is disabled.
+      this._secondField.focus();
```

Comment:
I _think_ there's been some change in product requirements here (i.e., "Website address" is always editable).  cc @sandysage for confirmation.

If so, I would propose a follow-up issue; there be dragons if this field is editable.

Reply:
discussed with @sandysage , and there is a change that had not been reflected into the desktop designs yet (it is in the mobile CRUD designs).

I'm going to file a follow up for this; editable hostname has complications/concerns we need to make sure we understand.